client/core/core.module.js

```diff
@@ -3,6 +3,7 @@
   angular.module('app.core', [
     'app.messaging',
     'app.player-messenger',
+    'app.player-user',
```

Comment:
Think this will be resolved with a rebase to the current master... pretty sure player-user is already in that list now